zer0-os/zOS
# Add address bar

### What does this do?
Adds an address bar component.

### Why are we making this change?
It is a requested feature.

### How do I test this?
Run the tests.
Open the app, look at the address bar. Follow a link. The address bar should update. Click a previous segment of the route. You should go back to that point in the route.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
